kinoml/core/measurements.py

```diff
@@ -118,17 +118,17 @@ def check(self):
         assert (0 <= self.values <= 100).all(), "One or more values are not in [0, 100]"
 
     @staticmethod
-    def _observation_model_pytorch(dG_over_KT, inhibitor_conc=1e-6, **kwargs):
+    def _observation_model_pytorch(dG_over_KT, inhibitor_conc=1, **kwargs):
         # FIXME: this might be upside down -- check!
         import torch
 
-        return 100 * (1 - 1 / (1 + inhibitor_conc / torch.exp(dG_over_KT)))
+        return 100 * (1 - 1 / (1 +1  / torch.exp(dG_over_KT)))
```

Comment:
Why do we need to hardcode this? Don't we have the argument in the function? Also, why are we changing units now?

Reply:
I need to speak with John for this observation model. I thought that this measurement type was in the clear, but I realized at the end of last week that something is fishy and I need to loop in John asap.